tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -563,6 +594,17 @@ describe(__filename, () => {
       .toHaveProp('header', 'Themes by these artists');
   });
 
+  it('shows extensions in header for a static theme with multiple authors and without More text ', () => {
+    const root = renderAddonsWithType({
+      addonType: ADDON_TYPE_STATIC_THEME,
+      showMore: false,
+      multipleAuthors: true,
+    });
+
+    expect(root.find(AddonsCard))
+      .toHaveProp('header', 'Themes by these artists');
+  });
```

Comment:
Thanks for adding coverage here, I wonder why it was not done before since you did not make any changes to `AddonsByAuthorsCard` in this patch.

Reply:
I *think* it was here before.. Could it have gotten lost in a rebase?

Reply:
mmh, interesting. Maybe GitHub shows the wrong diff. No worries, we'll see right before you merge this one :)